src/AppProviders.tsx

```diff
@@ -59,21 +59,21 @@ export function AppProviders({ children }: ProvidersProps) {
   return (
     <HelmetProvider>
       <ReduxProvider store={store}>
-        <PluginProvider>
-          <ColorModeScript storageKey='ss-theme' />
-          <ChatwootWidget />
-          <ChakraProvider theme={theme} colorModeManager={manager} cssVarsRoot='body'>
-            <ToastContainer />
-            <PersistGate loading={splashScreen} persistor={persistor}>
-              <HashRouter basename='/'>
-                <ScrollToTop />
-                <BrowserRouterProvider>
-                  <I18nProvider>
-                    <WalletProvider>
-                      <WalletConnectV2Provider>
-                        <KeepKeyProvider>
-                          <ErrorBoundary FallbackComponent={ErrorPage} onError={handleError}>
-                            <QueryClientProvider>
+        <QueryClientProvider>
```

Comment:
Moves `<QueryClientProvider />` up top, noticed sends were *crashing* the app for wcv2

Reply:
I was a bit afraid about this one, maybe it could cause conflicts with other query clients or any other libraries but I couldn't detect any bugs with this so, looks good so far

Reply:
Providers mang 😢 fix one bug, introduce another! I wasn't able to spot anything weird with this either but there is definitely a risk here, though there's a *known* bug keeping it where it is